warning: implicit declaration of function 'pfx_table_notify_diff'

```
tests/test_pfx.c:444:2: warning: implicit declaration of function
'pfx_table_notify_diff'; did you mean 'pfx_table_validate'?
[-Wimplicit-function-declaration]
```

--- a/tests/test_pfx.c
+++ b/tests/test_pfx.c
@@ -18,6 +18,7 @@
 #include "rtrlib/lib/ip_private.h"
 #include "rtrlib/lib/utils_private.h"
 #include "rtrlib/pfx/pfx.h"
+#include "rtrlib/pfx/pfx_private.h"
 #include "rtrlib/pfx/trie/trie_private.h"
 #include "rtrlib/rtr/rtr.h"
 

